cluster-up/cluster/kind-k8s-sriov-1.17.0/provider.sh

```diff
@@ -3,9 +3,20 @@
 set -e
 
 export CLUSTER_NAME="sriov"
-export KIND_NODE_IMAGE="kindest/node:v1.17.0"
+export K8S_VERSION="${K8S_VERSION:-1.17.0}"
 
-source ${KUBEVIRTCI_PATH}/cluster/kind/common.sh
+function set_kind_params() {
+    local k8s_version=$1
+
+    if [ "$k8s_version" == "1.17.0" ]; then
+        export KIND_NODE_IMAGE="${KIND_NODE_IMAGE:-kindest/node:v1.17.0}"
```

Comment:
Makes sense to re-use the `K8S_VERSION` variable, no?

Reply:
Tried, didnt managed yet how to use $k8s_version within the "${...}"
will resreach